telemetry/opting-out-of-telemetry.adoc

```diff
@@ -0,0 +1,12 @@
+:_content-type: ASSEMBLY
+[id="opting-out-of-telemetry"]
+= Opting out of Telemetry
+include::modules/common-attributes.adoc[]
+:context: logs-and-pod-status
+
+toc::[]
+
+[role="_abstract"]
+You must comply with privacy laws to opt out of Telemetry.
```

Comment:
```suggestion
You might need to opt out of Telemetry for your cluster. For example, you might need to comply with privacy laws.
```

Reply:
Telemetry data is anonymous, there is no PI collected, so I doubt there's a privacy concern here. The current message reads like actually everybody would prefer to opt-out due to privacy laws.

One reason to opt-out might be some internal regulation, which doesn't permit telemetry data traffic. Or some network policy. Or if a customer believes the collected data will be of no use for us (e.g., local development setup or some specific testing environment).

Reply:
How about this
> "The decision to opt out of telemetry should be based on your specific needs and requirements, as well as any applicable regulations or policies that you need to comply with." 

@0x656b694d